sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -195,15 +201,15 @@ def run_job(self):
 
         # error checking
         # prevent against backwards crawling
-        if '..' in semester or '..' in course or '..' in gradeable or '..' in config_id:
+        if '..' in semester or '..' in course or '..' in gradeable:
```

Comment:
This does not have the same cast to `int` on `config_id` as the `RunLichen` job does.